### PR TITLE
Fixed: remove docstrings from click text messages #775 and worked on #774

### PR DESCRIPTION
**Pull request by edeediong**

### CODE REVIEW

**Comment by edeediong** [comment]
> thanks for working on this @edeediong. Couple of comments:
> 
> * the docstrings need to be removed from the messages, but not from the comments. We can leave them in the form of `#` comments
> * only the docstring of `cli()` functions get shown by click, so these are the only functions we need to modify

Oh I misunderstood your previous instructions. Reverting and making changes now.

File: docs/sections/getting_started.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Getting Started
 
-Popper is a workflow execution engine based on [Github Actions][gha] 
-(GHA) written in Python. With Popper, you can execute [HCL 
-syntax][ghalang] workflows locally on your machine without having 
+Popper is a workflow execution engine based on [Github Actions][gha]
```
Comment (ivotron) [inline docs/sections/getting_started.md]: looks like this PR also contains the commits from the other PR you're also working on. Can you please remove them from this? or alternatively add both in a single PR and close one of them?

Comment (edeediong) [reply docs/sections/getting_started.md]: Alright I'd do exactly this

**Comment by ivotron** [comment]
thanks @edeediong !

**Comment by edeediong** [comment]
Glad I could contribute!

**Pull request merged**